labcodes/confetti-ds
# [CSYS-318] Migrating Tag components

Link to task: [Card from Jira](https://labcodes.atlassian.net/browse/CSYS-318)

Staging app:
https://CSYS-318-migrate-tag-components--confetti-storybook.netlify.app/

How to test:
Run the project locally and check if the Tag's subcomponents are working as expected.

Description of your solution:
Removing Class Components' methods and using Hooks to control the states and other properties.

Screenshots (when applicable):
N/A
